benchmarks/benchmark_suite/scripts/checkout_repos.bash

```diff
@@ -3,11 +3,11 @@
 # Kokkos
 if [ ! -d "kokkos" ]; then
   git clone https://github.com/kokkos/kokkos
+  cd kokkos
+  git checkout develop
+  git pull
+  cd ..
 fi
```

Comment:
Why did you change?  The intent seemed to be to pull the latest version when the repo had already been cloned

Reply:
True, that's a change of behavior, but I thought it would be more convenient. You don't need to modify the script when you are running the benchmark on a branch and no surprising pulls happen.

Reply:
That's a bit different from the other libraries in the script, but I think it should be fine since it's Kokkos that we are benchmarking here.